app/scripts/services/aggregatedLoggingService.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+angular.module("openshiftConsole")
```

Comment:
@spadgett angulars DI engine cannot find this service - do I need to manually register it somewhere?

I did run `hack/clean-deps.sh && hack/install-deps.sh && grunt build`

Reply:
I see that other services from app/scripts/services are in dist/scripts/scripts.js, but aggregatedLoggingService is not among them.